pom.xml

```diff
@@ -1461,7 +1461,7 @@
                             <plugin>
                                 <groupId>org.codehaus.mojo</groupId>
                                 <artifactId>findbugs-maven-plugin</artifactId>
-                                <version>2.4.0</version>
+                                <version>3.0.5-SNAPSHOT</version>
```

Comment:
patched in https://github.com/Jasig/uPortal/pull/865/commits/17c7a42bef0321267de2d416456553e37207a69b